exasol/toolbox/util/dependencies/shared_models.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import annotations
+
+from pydantic import BaseModel
+
+
+class Package(BaseModel):
+    name: str
+    version: str
```

Comment:
Would it be better to use https://packaging.pypa.io/en/latest/version.html#packaging.version.Version here instead of str?